csm/core/controllers/view.py

```diff
@@ -180,6 +181,12 @@ def get_extension(filename):
         parse_results = {}
 
         reader = await request.multipart()
+        
+        try:
+            FSUtils.create_dir(file_cache.cache_dir)
```

Comment:
Please create directory with proper umask set so that dir gets created with required permisssions.

Reply:
> Please create directory with proper umask set so that dir gets created with required permisssions.

Done. Please verify

Reply:
We should check if cache_dir does not exist then only it should be created.

Reply:
This check is already covered in FSUtills.create_dir() method.
@staticmethod
    def create_dir(path: str):
        """
        Create directory by a given path. All sub-directories are created recursively

        :param path: path to directory which need to create
        :return:
        """
        if not os.path.exists(path):
            try:
                os.makedirs(path)
            except OSError as e:
                raise CsmInternalError(f"System error during directory creation for "
                                       f"path='{path}': {e}")
        else:
            raise ResourceExist(f"Can not create directory {path}: it already exists ")